app/views/user_sessions/_form.html.slim

```diff
@@ -0,0 +1,13 @@
+= form_tag user_sessions_path, :method => :post do
+  div class="field"
```

Comment:
It's a matter of preference, I'm used to reading markup this way so I prefer:
```suggestion
  .field
```
Totally up to you if you want to leave this more explicit.

Reply:
No, I've seen it done that way before. I guess I'm still getting comfortable with slim, but writing ".class" and "#id" maks more sense.